crates/matrix-sdk/src/widget/settings/element_call.rs

```diff
@@ -70,14 +79,38 @@ pub enum EncryptionSystem {
     },
 }
 
+/// Defines the intent of showing the call.
+///
+/// This controls whether to show or skip the lobby.
+#[derive(Debug, PartialEq, Serialize, Default)]
+#[serde(rename_all = "snake_case")]
+pub enum Intent {
+    #[default]
+    /// The user wants to start a call. This will skip the lobby.
```

Comment:
I don't think we can guarantee that it will skip the lobby. So, maybe it would be better to just say this?

```suggestion
    /// The user wants to start a call.
```

Reply:
I hoped this would help app devs to find what they need to use instead of the `skip_lobby` flag. but I suppose this doc has to happen somewhere else then.